en/05_Contributing/18_Deprecations.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Deprecations
```

Comment:
I don't think this belongs under `05_Contributing` - this is about peoples _own projects_.
Probably `03_Upgrading/02_Deprecations` would be appropriate. Since this is about deprecation warnings in one's own project during an upgrade.

Reply:
Opps.  I put it update 03_Upgrade/07_Deprecations - not 02 since there's already an 02 for "02_Upgrade_module"